cmd/go-cache-plugin/commands.go

```diff
@@ -26,6 +26,7 @@ var flags struct {
 	CacheDir      string        `flag:"cache-dir,default=$GOCACHE_DIR,Local cache directory (required)"`
 	S3Bucket      string        `flag:"bucket,default=$GOCACHE_S3_BUCKET,S3 bucket name (required)"`
 	S3Region      string        `flag:"region,default=$GOCACHE_S3_REGION,S3 region"`
+	UsePathStyle  bool          `flag:"path-style,default=$GOCACHE_PATH_STYLE,S3 path-style URLs (optional)"`
```

Comment:
Let's harmonize the naming here, I suggest:
```suggestion
	S3PathStyle  bool          `flag:"s3-path-style,default=$GOCACHE_S3_PATH_STYLE,S3 path-style URLs (optional)"`
```